lib/upload.js

```diff
@@ -276,13 +284,23 @@ class Upload {
 
     cb = cb || function () {};
     if (shouldTerminate && this.url != null) {
-      Upload.terminate(this.url, this.options, (err, xhr) => {
-        if (err) {
-          return cb(err, xhr);
-        }
+      const terminate = () => {
+        Upload.terminate(this.url, this.options, (err, xhr) => {
```

Comment:
The callback for `Upload.terminate` does not provide the second argument `xhr` if I am not mistaken. See https://github.com/tus/tus-js-client/blob/master/lib/upload.js#L93

Reply:
Hmm, seems I got confused by the brackets. But do you think we should be passing it as a matter of fact? 🤔 It suddenly feels unnecessary to have it since the xhr instance is always added to the error instance. Also removing it will not be a breaking change since it was never there.

Reply:
> But do you think we should be passing it as a matter of fact?

No, the XHR is already part of the error object if available, as you also said. Please also remove.